docs/faq/perf.md

```diff
@@ -117,37 +117,46 @@ and  [MXNet-1.2.0.rc1](https://github.com/apache/incubator-mxnet/releases/downlo
 
 - K80 (single GPU)
 
-| Batch | Alexnet | VGG    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+| Batch | Alexnet | VGG 16    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
 |-------|---------|--------|--------------|--------------|-----------|------------|
 | 1     | 243.93  | 43.59  | 68.62        | 35.52        | 67.41     | 23.65      |
 | 2     | 338.16  | 49.14  | 113.41       | 56.29        | 93.35     | 33.88      |
 | 4     | 478.92  | 53.44  | 159.61       | 74.43        | 119.18    | 45.23      |
 | 8     | 683.52  | 70.50  | 190.49       | 86.23        | 131.32    | 50.54      |
 | 16    | 1004.66 | 109.01 | 254.20       | 105.70       | 155.40    | 62.55      |
 | 32    | 1238.55 | 114.98 | 285.49       | 116.79       | 159.42    | 64.99      |
+| 64 | 1346.72 | 123.56 | 308.73 | 122.21 | 167.58 | 70.21 |
+| 128 | 1416.91 | OOM | 320.98 | 123.11 | 171.55 | 71.85 |
+| 256 | 1462.97 | OOM | 329.16 | 127.53 | 153.01 | 57.23 |
 
 - M60
 
-| Batch | Alexnet | VGG    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+| Batch | Alexnet | VGG 16    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
 |-------|---------|--------|--------------|--------------|-----------|------------|
 | 1     | 243.49  | 59.95  | 101.97       | 48.30        | 95.46     | 39.29      |
 | 2     | 491.04  | 69.14  | 170.35       | 80.27        | 142.61    | 60.17      |
 | 4     | 711.54  | 78.94  | 257.89       | 123.09       | 182.36    | 76.51      |
 | 8     | 1077.73 | 109.34 | 343.42       | 152.82       | 208.74    | 87.27      |
 | 16    | 1447.21 | 144.93 | 390.25       | 166.32       | 220.73    | 92.41      |
 | 32    | 1797.66 | 151.86 | 416.69       | 176.56       | 230.19    | 97.03      |
+| 64 | 1779.38 | 150.18 | 427.51 | 183.47 | 239.12 | 101.59 |
+| 128 | 1787.36 | OOM | 439.04 | 185.29 | 243.31 | 103.39 |
+| 256 | 1899.10 | OOM | 450.22 | 183.42 | 242.36 | 100.98 |
 
 
 - V100
 
-| Batch | Alexnet | VGG    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
+| Batch | Alexnet | VGG 16    | Inception-BN | Inception-v3 | Resnet 50 | Resnet 152 |
 |-------|---------|--------|--------------|--------------|-----------|------------|
-| 1     | 659.51  | 205.16 | 136.91       | 76.54        | 162.15    | 61.38      |
-| 2     | 1248.21 | 265.40 | 261.85       | 144.23       | 293.74    | 116.30     |
-| 4     | 2122.41 | 333.97 | 477.22       | 270.03       | 479.14    | 195.17     |
-| 8     | 3894.30 | 420.26 | 831.09       | 450.68       | 699.39    | 294.19     |
-| 16    | 5815.58 | 654.16 | 1332.26      | 658.97       | 947.45    | 398.79     |
-| 32    | 7906.09 | 708.43 | 1784.23      | 817.33       | 1076.81   | 451.82     |
+| 1     | 659.51  | 205.16 | 157.37 | 87.71 | 162.15    | 61.38      |
```

Comment:
Are you sure this would render correctly? Do you need to align the columns?

Reply:
this is generated by typora. it should be fine.